SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -107,5 +113,29 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	/** Pack unreliable RPCs sent during the same frame into a single update. */
 	UPROPERTY(config, meta = (ConfigRestartRequired = false))
 	bool bPackUnreliableRPCs;
-};
 
+	UPROPERTY(EditAnywhere, Config, Category = "Offloading")
+	bool bEnableOffloading;
+
+	/** Map of Actor Group to set of Actor classes it contains. */
+	UPROPERTY(EditAnywhere, Config, Category = "Offloading", meta = (EditCondition = "bEnableOffloading"))
+	TMap<FName, FActorClassSet> ActorGroups;
+
+	/** Set of Managed worker types to run. */
+	UPROPERTY(EditAnywhere, Config, Category = "Offloading", meta = (EditCondition = "bEnableOffloading"))
+	TSet<FName> WorkerTypes;
```

Comment:
This being a set causes some ordering issues in the UI - when you have a few items and add a new one it sometimes appears in between them.

Reply:
Does this list only contain server workers? if so I think we should rename it to ServerWorkerTypes to be more explicit.